setup.py

```diff
@@ -2,36 +2,43 @@
 
 from setuptools import setup, find_packages
 
-install_requires = [
+INSTALL_REQUIRES = [
     'lxml',
     'setuptools',
-    'Django>=1.4',
+    'Django>=1.4,<1.7',
     'django-appconf',
     'django-cms>=2.4',
     'django-classy-tags>=0.3.2',
-    'django-haystack>=2.0.0',
+    'django-haystack>=2.4.0.dev0',
```

Comment:
I agree with you on that @czpython, I'm checking with django-haystack maintainer (Chris Adams) to apply that for 2.3.x versions. If he does not I probably will implement it in aldryn-search itself. I used 2.4.0.dev0 here because I tested it extensively, but since this PR depends of django-haystack PR I will change this depending of what is done in django-haystack.